src/services/routes/org.routes.ts

```diff
@@ -162,5 +161,5 @@ export default function registerRoutes(
     },
   });
 
-  done();
+  await fastify.register(memberRoutes, { ...opts, prefix: '/orgs/:orgName/members' });
```

Comment:
I suggest having a dedicated file for the route prefixes. this way it would be way easier to control the prefixes and check errors in case we have any
For example 
```
const ORG_PREFIX = "/orgs"
const MEMBER_PREFIX = "/orgs/:orgName/members"
```

Reply:
> this way it would be way easier to control the prefixes and check errors in case we have any

Can you please elaborate on that?

Unless I made a mistake somewhere, those values are only ever used once, so moving them to a separate file adds complexity and I'm not sure I see the benefit(s).